engine/class_modules/sc_rogue.cpp

```diff
@@ -4125,9 +4125,9 @@ struct shiv_t : public rogue_attack_t
     }
   }
 
-  // 1/23/2021 - Does not appear to proc Combat Potency despite being an OH attack
+  // 06/24/2021 - Now procs combat potency according to 9.1 Patch Notes
```

Comment:
```suggestion
  // 2021-06-24 - Now procs combat potency according to 9.1 Patch Notes
```
https://github.com/simulationcraft/simc/wiki/CodingGuidelines#basic-rules

Reply:
fixed it, thank you!